procmon/processmonitor.cpp

```diff
@@ -3128,7 +3103,7 @@ int ProcessMonitor::updateLog(std::string action, std::string level)
     //if non-root, change file permissions so we can update it
     if ( !rootUser)
     {
-        string cmd = "sudo chmod 666 " + fileName + " > /dev/null";
+        string cmd = "chmod 666 " + fileName + " > /dev/null 2>&1";
```

Comment:
we shouldn't make the log config world-writable.  Also this won't work as not-root.